src/components/ErrorsManager/ErrorsManager.tsx

```diff
@@ -0,0 +1,31 @@
+import { FC } from 'react';
+import { Errors } from '../../App';
+import classNames from 'classnames';
+
+interface Props {
+  error: Errors;
+  errorHide: () => void;
+}
+
+export const ErrorsManager: FC<Props> = ({ error, errorHide }) => {
+  return (
+    <div
+      data-cy="ErrorNotification"
+      className={classNames(
+        'notification is-danger is-light has-text-weight-normal',
+        {
+          hidden: error === Errors.default,
+        },
+      )}
+    >
+      <button
+        data-cy="HideErrorButton"
+        type="button"
+        className="delete"
+        title="Hide Error"
+        onClick={errorHide}
+      ></button>
+      {error}
```

Comment:
when errors fade away (after timeout or close on button), they change to 'No errors'
![image](https://github.com/mate-academy/react_todo-app-loading-todos/assets/147381375/fe572b2c-c091-406c-b2df-976287993295)